lib/core/src/Cardano/Wallet/Primitive/Mnemonic.hs

```diff
@@ -214,6 +253,7 @@ fromUtf8String = T.pack . Basement.toList
 
 instance (KnownNat csz) => Basement.Exception (MnemonicException csz)
 
+-- | Convert a 'Mnemonic' to a sentence of English mnemonic words.
```

Comment:
I wonder if more correct would be here `-- | Convert a 'Mnemonic' to a sentence of mnemonic English words.` @jonathanknowles @rvl ?

Reply:
> I wonder if more correct would be here `-- | Convert a 'Mnemonic' to a sentence of mnemonic English words.` @jonathanknowles @rvl ?

Well, it probably isn't a "sentence" unless it obeys the rules of English grammar. I'd probably use "string of English words", or something similar.